### DISCUSSION TITLE
Welcome to χrust Discussions!

### DISCUSSION THREAD

**Discussion by ballsteve** [ballsteve/xrust#18] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Participate in setting the direction of the χrust project.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ballsteve** [comment]
Hi, I'm Steve. I started the χrust project.

I'm working mainly on the XPath and XSLT modules.

**Comment by Devasta** [comment]
Hi Steve!

I'm Daniel, I'm an AVP in a hedge fund admin, and have been using XSLT (alas, only 1.0) for about 7 years now to manage reporting of clients trades to their brokers and custodians.

My current focus is on the XML parser.
